src/helpers.rs

```diff
@@ -43,11 +44,32 @@ pub(crate) fn write_to_file<'a>(path: &Path, contents: &'a str) {
 }
 
 /// Clone `url` into `target` and degit it
-pub(crate) fn clone_and_degit(url: &str, target: &Path) -> Result<()> {
+pub(crate) fn clone_and_degit(url: &str, target: &Path) -> Result<String> {
 	let repo = Repository::clone(url, target)?;
+
+	// fetch tags from remote
+	let release = fetch_tag(&repo);
+
 	let git_dir = repo.path();
 	fs::remove_dir_all(&git_dir)?;
-	Ok(())
+	Ok(release)
+}
+
+/// Fetch the latest release from a repository
+fn fetch_tag(repo: &Repository) -> String {
+	let mut release = String::new();
+
+	let version_reg = Regex::new(r"v\d+\.\d+\.\d+").unwrap();
+
+	for tag in repo.tag_names(None).unwrap().iter() {
```

Comment:
So I'd like to add here, that while this works, `fetch_tag` needs to be Optional (`Option<String>`) and that should be enough of a change to remove all instances of `unwrap` here. Template generation should not fail because `fetch_tag` failed. I believe this should've been in the issue description rather than here, but the problem is, if in future, we decide to support a template that doesn't follow the regex `"v\d+\.\d+\.\d+"`, then this code will also be modified so it's like a pending todo.